deploy/docker-swarm/README.md

```diff
@@ -0,0 +1,97 @@
+# Docker Swarm Hetzner CSI plugin
+
+Currently in Beta. Please consult the Docker Swarm documentation
+for cluster volumes (=CSI) support at https://github.com/moby/moby/blob/master/docs/cluster_volumes.md
+
+## How to install the plugin
+
+Run the following steps on all nodes (especially master nodes).
+The simplest way to achieve this
+
+1. Install the plugin 
+
+```bash
+docker plugin install --disable --alias hetznercloud/csi-driver-docker --grant-all-permissions hetznercloud/csi-driver-docker
+```
+
+2. Set HCLOUD_TOKEN
```

Comment:
Could you describe how the user needs to create the token and what permissions are required?

See in our kubernetes guide: https://github.com/hetznercloud/csi-driver/blob/e445d79e0a22fc5637e8862e76e0160bb20983ff/README.md?plain=1#L12

Reply:
I added a similar line